modules/virt-hot-plugging-disks-ui.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+:_content-type: PROCEDURE
+[id="virt-hot-plugging-disks-ui_{context}"]
+= Hot plugging and hot unplugging a disk by using the web console
```

Comment:
In theory, hot plug and hot unplug should be separate modules, as they are different use cases.